client/src/components/Signup.js

```diff
@@ -1,36 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { setAlert } from '../actions/alert';
+import PropTypes from 'prop-types';
+
+// Material-UI
 import Avatar from '@mui/material/Avatar';
 import Button from '@mui/material/Button';
 import CssBaseline from '@mui/material/CssBaseline';
 import TextField from '@mui/material/TextField';
 import Link from '@mui/material/Link';
 import Box from '@mui/material/Box';
 import Grid from '@mui/material/Grid';
-// import Checkbox from '@mui/material/Checkbox';
-// import FormControlLabel from '@mui/material/FormControlLabel';
 import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
-import axios from 'axios';
 
 const theme = createTheme();
 
-function Signup() {
-	const handleSubmit = async event => {
-		event.preventDefault();
+const Signup = ({ setAlert }) => {
+	const [formData, setFormData] = useState({
+		userName: '',
+		email: '',
+		password: '',
+		password_confirm: '',
```

Comment:
added another text field for password so we can make sure the user matches their password when signing up